docs/cloud-workload-security/agent_expressions.md

```diff
@@ -218,6 +218,7 @@ The *file.rights* attribute can now be used in addition to *file.mode*. *file.mo
 | `process.ancestors.fsuser` | string | FileSystem-user of the process |  |
 | `process.ancestors.gid` | int | GID of the process |  |
 | `process.ancestors.group` | string | Group of the process |  |
+| `process.ancestors.is_kworker` | bool |  |  |
```

Comment:
Could you confirm if it's expected for these new event types to lack descriptions? All other event types in the table include them.